example/standalone_git/README.md

```diff
@@ -0,0 +1,26 @@
+# NiFi Registry with GIT version control integration
+
+This example deploys a NiFi Registry container with git integration (Github). 
+Before running the example, these variables must be set beforehand. 
+
+| Variable   | Default value | Description |
+| ------------- | ------------- | ------------- |
+| `git_remote_url`  | (empty)  | URL of the remote git repository. Must be HTTPS. |
+| `git_checkout_branch`  | (empty)  |Branch to checkout and track. If none is specified, repository is only cloned and no branch switching is done.|
```

Comment:
This variable can't be optional/empty. It must have a value otherwise the container fails whith missing repository for git_flow_storage_directory. The text should be changed

Reply:
The column `Default value` shows what the value is if the user doesn’t write anything, but it doesn’t mean that the variabel is optional. The text above the table specifies that the these variable must be set beforehand. If I understand you correctly , do you mean that this should be even more specified ?